src/components/home/LandingPage.tsx

```diff
@@ -9,10 +9,13 @@ import LandingFooter from "./LandingFooter";
 import useMobileWarning from "@/hooks/useMobileWarning";
 import MobileWarningModal from "../mobileUnsupported/MobileWarningModal";
 import { signIn } from "next-auth/react";
+import BossBanner from "../banner/BossBanner";
+import { useRouter } from "next/router";
 
 const LandingPage = () => {
   const { isMobileAction, showWarning, closeWarning } = useMobileWarning();
-
+  const router = useRouter();
+  const isHomePage = router.pathname === "/";
```

Comment:
This is the home page by definition, so you shouldn't need this check

Reply:
A name like isRoot will suffice because landingPage component is used in  / and  /home for loggedin users, but it's not needed that was the first solution that came to mind, moved the banner out